baseline_testing/testcase_bestest_air_scenario.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suggest one script called something like ``run_baselines.py`` that can be used to simulate all test cases, instead of ``testcase_<name>_scenario.py`` for each test case.  Thereby removing the large amount of duplicated code for each test case.  I suggest the ``main`` of this script specify a list of the test cases to run.  I also suggest creating a configuration file, maybe ``config.json`` that specifies the scenarios to run for each test case (e.g. what is currently defined in lines 101-102 here).  So instead of those lines, the script can load the config file and read for the given test case which scenarios to run.

Reply:
Edit to this comment:  Rather than specify a list of test cases in ``main``, I suggest adding the test case name as a command line argument.  This way, the makefile can be used to run all the tests, including starting up and shutting down the boptest test case docker containers.